lib/rwssmg.gd

```diff
@@ -38,7 +38,7 @@
 ##  an attribute storing object (see <Ref Sect="Representation"/>).
 ##  <P/>
 ##  <Example><![CDATA[
-##  gap> f := FreeSemigroup( "a" , "b" );;
+##  gap> f := FreeSemigroup( "a", "b" );;
 ##  gap> a := GeneratorsOfSemigroup(f)[1];;
```

Comment:
Indeed.  Commands of the form "a := GeneratorsOfSemigroup(f)[1];" appear throughout the examples in this chapter.  I will make some more changes.